File: Proyecto en C#/Entrega Diseno/Banda.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Entrega_Diseno
{
    class Banda
    {
        private string _Nombre;
        public string Nombre
        {
            get { return _Nombre; }
            set { _Nombre = value; }
        }
        private CategoriaBanda _Categoria;
        public CategoriaBanda Categoria
        {
            get { return _Categoria; }
            set { _Categoria = value; }
        }
    }
}

File: recital-dominio/src/main/scala/recital/Banda.scala
package recital

import uqbar.arena.persistence.annotations.PersistentClass
import org.uqbar.commons.model.Entity
import uqbar.arena.persistence.annotations.PersistentField
import uqbar.arena.persistence.annotations.Relation
import compatibility.DomainEntity

@PersistentClass
class Banda extends DomainEntity {
	def this(nom:String, cat:CategoriaBanda) = {
	  this()
	  this.nombre(nom)
	  this.categoria (cat)
	}
  
	var _nombre:String = _
	var _categoria:CategoriaBanda = _
	
	def nombre(n:String) = _nombre = n
	@PersistentField
	def nombre = _nombre
	
	
	def categoria(c:CategoriaBanda) = _categoria = c
	@Relation
	def categoria = _categoria
	
	  
	def precioCategoria : Double = categoria.precioExtra
}